cmd/gardener-resource-manager/app/app.go

```diff
@@ -194,7 +206,22 @@ func NewResourceManagerCommand() *cobra.Command {
 		rootCAControllerOpts,
 		projectedTokenMountWebhookOpts,
 	)
+	flags := cmd.Flags()
+	opts.AddFlags(flags)
 	verflag.AddFlags(cmd.Flags())
 
 	return cmd
 }
+
+type Options struct {
+	// EnableProfiling enables profiling via web interface host:port/debug/pprof/.
+	EnableProfiling bool
+	// EnableContentionProfiling enable lock contention profiling, if profiling is enabled
+	EnableContentionProfiling bool
+}
+
+// AddFlags adds gardener-resource-manager's flags to the specified FlagSet.
+func (o *Options) AddFlags(fs *pflag.FlagSet) {
+	fs.BoolVar(&o.EnableProfiling, "profiling", true, "Enable profiling via web interface host:port/debug/pprof/")
```

Comment:
Why it is enabled by default when the same setting is disabled for all components except GAPI?

```suggestion
	fs.BoolVar(&o.EnableProfiling, "profiling", false, "Enable profiling via web interface host:port/debug/pprof/")
```

Reply:
I was testing, forgot to make it false again.